command/agent/config.go

```diff
@@ -434,6 +434,15 @@ type Telemetry struct {
 	// key/value structure as done in older versions of Nomad
 	BackwardsCompatibleMetrics bool `mapstructure:"backwards_compatible_metrics"`
 
+	// PrefixFilter allows for filtering out metrics from being collected
```

Comment:
Don't we need: https://www.consul.io/docs/agent/options.html#telemetry-filter_default

Reply:
This defaults to true in the metrics config, we can expose it though. I think it makes sense to.

Reply:
Added FilterDefault